szperacz/error_log.txt

```diff
@@ -0,0 +1,4 @@
+[ERROR] start error
```

Comment:
Do not put generated files into the repository.

Please add this file to gitignore:

https://github.com/vendral/szperacz/blob/main/.gitignore